app/views/embed/body/_file.html.erb

```diff
@@ -0,0 +1,83 @@
+<div class='sul-embed-body sul-embed-file'
+      data-sul-embed-theme="<%= viewer.asset_url('file.css') %>"
```

Comment:
Hmm.. maybe not? https://github.com/sul-dlss/sul-embed/blob/master/app/viewers/embed/viewer/common_viewer.rb#L15